docs/project-model/project-json-reference.md

```diff
@@ -0,0 +1,847 @@
+* [analyzerOptions](#analyzerOptions)
+    * [languageId](#languageId)
+* [configurations](#configurations)
+* [frameworks](#frameworks)
+    * [dependencies](#dependencies)
+    * [frameworkAssemblies](#frameworkAssemblies)
+    * [wrappedProject](#wrappedProject)
+    * [imports](#imports)
+    * [bin](#bin)
+        * [assembly](#assembly)
+
+<a name="name"></a>
+## name
+Type: String
+
+The name of the project, used for the assembly name as well as the name of the package. The top level folder name is used if this property is not specified.
+
+For example:
+
+    {
+        "name": "MyLibrary"
+    }
+
+<a name="version"></a>
+## version
+Type: String
+
+The [Semver](http://semver.org/spec/v1.0.0.html) version of the project, also used for the nuget package.
+
+For example:
+
+    {
+        "version": "1.0.0-*"
+    }
+
+<a name="summary"></a>
+## summary
+Type: String
+
+A short description of the project.
+
+For example:
+
+    {
+        "summary": "This is my library."
+    }
+
+<a name="description"></a>
+## description
+Type: String
+
+A longer description of the project. Used in the assembly properties.
+
+For example:
+
+    {
+        "description": "This is my library and it's really great!"
+    }
+
+<a name="copyright"></a>
+## copyright
+Type: String
+
+The copyright information for the project. Used in the assembly properties.
+
+For example:
+
+    {
+        "copyright": "Fabrikam 2016"
+    }
+
+<a name="title"></a>
+## title
+Type: String
+
+The friendly name of the project, can contain spaces and special characters not allowed when using the `name` property. Used in the assembly properties.
+
+For example:
+
+    {
+        "title": "My Library"
+    }
+
+<a name="entryPoint"></a>
+## entryPoint
+Type: String
+
+The entrypoint method for the project. `Main` by default.
+
+For example:
+
+    {
+        "entryPoint": "ADiffrentMethod"
+    }
+
+<a name="projectUrl"></a>
+## projectUrl
+Type: String
+
+The url for the homepage of the project.
+
+For example:
+
+    {
+        "projectUrl": "http://www.mylibrary.gov"
+    }
+
+<a name="licenseUrl"></a>
+## licenseUrl
+Type: String
+
+The url for the licence the project uses.
+
+For example:
+
+    {
+        "licenseUrl": "http://www.mylibrary.gov/licence"
+    }
+
+<a name="iconUrl"></a>
+## iconUrl
+Type: String
+
+The url for an icon that will be used in various places such as the package explorer.
+
+For example:
+
+    {
+        "iconUrl": "http://www.mylibrary.gov/favicon.ico"
+    }
+
+<a name="compilerName"></a>
+## compilerName
+Type: String
+
+The name of the compiler used for this project. `csc` by default. currently `csc`, the c# compiler or `fsc`, the f# compiler are supported.
+ 
+For example:
+
+    {
+        "compilerName": "fsc"
+    }
+
+<a name="testRunner"></a>
+## testRunner
+Type: String
+
+The name of the testrunner, such as nuint or xunit, to use with this project, setting this also marks the project as a test project.
```

Comment:
I know that xUnit has a page, but now that I think about it it's still DNX-based.  Probably better to link to the home pages for now.